src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invented = {};
```

Comment:
It's not clear what should be stored in this variable. Let's think more about meaningful and clear variable name ⏰